terraform/Makefile

```diff
@@ -0,0 +1,41 @@
+# Define variables for Terraform
+TF      = terraform
+TF_MAIN_DIR  = mainnet
+TF_CALIB_DIR = calibnet
+
+# Define the default target
+.DEFAULT_GOAL := help
+
+help:
+	@echo "Usage: make [target]"
+	@echo ""
+	@echo "Targets:"
+	@echo "  init_calib or init_main      	Initialize Terraform for either mainnet or calibnet"
+	@echo "  plan_calib or plan_main     	Generate and show an execution plan for either mainnet or calibnet"
+	@echo "  apply_main or apply_calib  	Apply the changes for either mainnet or calibnet"
+	@echo "  destroy_main or destroy_calib  Destroy the Terraform-managed infrastructure for either mainnet or calibnet"
+	@echo "  help      						Shows this help message"
```

Comment:
You're mixing tabs and spaces. Always use spaces. Otherwise, the indentation will not be the same for everyone.